synapse/config/server.py

```diff
@@ -490,6 +496,13 @@ def default_config(self, server_name, data_dir_path, **kwargs):
 
         # Used by phonehome stats to group together related servers.
         #server_context: context
+
+        # Resource-constrained Homeserver Settings
+        #
+        # Requires _joins to be set to True.
+        #
+        #limit_large_remote_room_joins: False
```

Comment:
```suggestion
        #limit_large_remote_room_joins: True
```
so you can say "uncomment the below to enable"

Reply:
done